app/components/small-cart-toggle-view.js

```diff
@@ -2,10 +2,21 @@ import Ember from "ember";
 
 export default Ember.Component.extend({
   showCartSummary: false,
+  allowClick: false,
+
+  application: Ember.computed(function() {
+    return Ember.getOwner(this).lookup("controller:application");
+  }),
 
   actions: {
     toggleCartSummary() {
       this.toggleProperty("showCartSummary");
+    },
+    showItemDetails(item) {
+      if (!this.get("allowClick")) {
```

Comment:
Ammm what is the use of `allowClick` here?
The initial value if defaulting to false and I don't see it changing anywhere else in this component.
So `!this.get("allowClick")` will be always true

Reply:
Also that property is not passed to component either *{{{{small-cart-toggle-view}}}}*

Reply:
It's for the ability to turn it on later on. We had a long discussion with Matt on whether we wanted to allow user to navigate away from the order process in the middle (by clicking on an item)

Reply:
We can remove it if you don't like having an un-used feature, I don't mind

Reply:
I wasn't aware about the discussion with Matt 😄 
I don't mind keeping un-used features but it just hard to understand at first glance.
Can you add a similar comment so it'll be easier to understand for everyone?